pkg/backend/aio_validate.go

```diff
@@ -0,0 +1,64 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright (c) 2022-2023 Dell Inc, or its subsidiaries.
+
+// Package backend implememnts the BackEnd APIs (network facing) of the storage Server
+package backend
+
+import (
+	"go.einride.tech/aip/fieldbehavior"
+	"go.einride.tech/aip/resourceid"
+	"go.einride.tech/aip/resourcename"
+
+	pb "github.com/opiproject/opi-api/storage/v1alpha1/gen/go"
+)
+
+func (s *Server) validateCreateAioVolumeRequest(in *pb.CreateAioVolumeRequest) error {
```

Comment:
I am thinking if we could export those methods? Those methods just verify the common rules for gRPC request fields, and we could reuse it in other bridges

Reply:
I was thinking we should add more tests now that it is in separate function
for example for AIO, we can test if file exists... block size is modulo 512
and so on...
once we do it, not sure how much common we will end up...
but generically code reduction is great

Reply:
ok, let's see, but we could separate the common logic into a separate method and the logic specific for a given bridge into another